src/views/Home.vue

```diff
@@ -70,8 +78,28 @@ export default class Home extends Vue {
       this.$router.push("Results");
     });
 
+    this.Survey.onAfterRenderPage.add(result => {
+      var progressBar = document.getElementsByClassName("progress-bar")[0];
+      //Make sure that the current page is 0 and the progress bar is defined and displayed on the screen
+      if (result.currentPageNo == 0 && progressBar != undefined) {
+        progressBar.innerHTML = "Page 1 of 13";
+      }
+      //Increment the current page by one
+      else {
+        if (progressBar != undefined) {
+          progressBar.innerHTML =
+            "Page " + (result.currentPageNo + 1) + " of 13";
```

Comment:
Page of... needs to be translated to French. Switch on locale, and use "Page ... de 13"